ui/shared/status/StatusChatInput.qml

```diff
@@ -58,6 +58,10 @@ Rectangle {
         muted: !appSettings.notificationSoundsEnabled
     }
 
+    function isStatusUpdateInput() {
+        return chatType === Constants.chatTypeStatusUpdate
+    }
```

Comment:
You can set this as a property instead, so that the comparison is only done once. Also, you don't have to put parentheses everywhere haha